lib/jekyll/filters/url_filters.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "addressable/uri"
+require "pathname"
```

Comment:
this line is not required as its already loaded by [`jekyll.rb`](https://github.com/jekyll/jekyll/blob/v3.5.2/lib/jekyll.rb#L24)